deployments/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -85,18 +85,18 @@ data:
     - name: {{ .Values.nsm.serviceName | quote }}
       labels:
         app: "vl3-nse-{{ .Values.nsm.serviceName }}"
-{{- if .Values.cnns.nsr.addr }}
-        cnns/nsr.addr: {{ .Values.cnns.nsr.addr | quote }}
-        cnns/nsr.port: {{ .Values.cnns.nsr.port | quote }}
+{{- if .Values.wcm.nsr.addr }}
+        wcm/nsr.addr: {{ .Values.wcm.nsr.addr | quote }}
+        wcm/nsr.port: {{ .Values.wcm.nsr.port | quote }}
 {{- end }}
-      cnns:
+      wcm:
         name: {{ .Values.nsm.serviceName | quote }}
-        address: "{{ .Values.cnns.nsr.addr }}"
+        address: "{{ .Values.wcm.nsr.addr }}"
         connectivityDomain: "{{ .Values.nsm.serviceName }}-connectivity-domain"
       vl3:
        ipam:
```

Comment:
Since you have changes in both this repo and the API one, you could change the name of this attribute to wcmd, and the server address from ipam-... to wcmd-... . Or simply have the whole address be passed as a parameter. So we don't have to change this in the future.

Reply:
Updated